tests/pipelines/stable_diffusion/test_stable_diffusion_repaint.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+# Copyright 2022 HuggingFace Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+import unittest
+
+import numpy as np
+import torch
+
+from diffusers import RePaintScheduler, StableDiffusionRepaintPipeline
+from diffusers.utils import load_image, slow, torch_device
+from diffusers.utils.testing_utils import load_numpy, require_torch_gpu
```

Comment:
As Patrick mentioned above, most of the models are getting covered by common tests that check API compatibility etc. 
What we need here is just a test class similar to `RepaintPipelineFastTests`: https://github.com/huggingface/diffusers/blob/f17fae641c7c268e60c479396891ea97a6b29aae/tests/pipelines/repaint/test_repaint.py#L31
with `pipeline_class = StableDiffusionRepaintPipeline` and slightly adapted `get_dummy_components()` and `get_dummy_inputs()` which you can probably borrow without many changes from `StableDiffusionInpaintPipelineFastTests`: https://github.com/huggingface/diffusers/blob/f17fae641c7c268e60c479396891ea97a6b29aae/tests/pipelines/stable_diffusion/test_stable_diffusion_inpaint.py#L46

Reply:
These added tests will probably uncover some missing pieces in the pipeline, so feel free to ping us if something is tough to fix! :)